web/packages/teleterm/src/ui/Search/searchResult.ts

```diff
@@ -48,6 +48,13 @@ export type SearchResultResourceType = {
   nameMatch: string;
   score: number;
 };
+export type DocumentClusterSearchResult = {
+  kind: 'document-cluster-search';
```

Comment:
We're in a tight spot here when it comes to naming. We're within `src/ui/Search`, showing search results inside a search bar, one of them being a search result which talks about showing search results in a new tab.

I was thinking of something like `display-results` or something along those lines?

Reply:
Right, the name wasn't prefect, changed to `display-results`.